source/languageHandler.py

```diff
@@ -101,21 +101,22 @@ def getAvailableLanguages(presentational=False):
 	if 'en' not in locales:
 		locales.append('en')
 		locales.sort()
-	#For each locale, ask Windows for its human readable display name
-	displayNames = []
-	for entry in locales:
-		desc=getLanguageDescription(entry)
-		displayNames.append("%s, %s"%(desc,entry) if desc else entry)
-	#Prepare a zipped view of language codes and descriptions.
-	# #7284: especially for sorting by description.
-	# Python 3: zip function changed from returning a list to an iterator, thus wrap this inside a list call.
-	langs = list(zip(locales,displayNames))
-	if presentational:
-		langs.sort(key=lambda lang: lang[1])
+	# Prepare a 2-tuple list of language code and human readable language description.
+	langs = [(lc, getLanguageDescription(lc)) for lc in locales]
+	# Translators: The pattern defining how languages are displayed and sorted in in the general
+	# setting panel language list. Use "{desc}, {lc}" (most languages) to display first full anguage

```

Comment:
missing L in language:
```suggestion
	# setting panel language list. Use "{desc}, {lc}" (most languages) to display first full language

```